cmd/list.go

```diff
@@ -208,13 +296,13 @@ func byteSizeToString(size int64) string {
 		"GiB",
 		"TiB",
 		"PiB",
-		"EiB", // Let's face it, a file, account, or container probably won't be more than 1000 exabytes in YEARS. (and int64 literally isn't large enough to handle too many exbibytes. 128 bit processors when)
+		"EiB", // Let's face it, a file, account, or container probably won't be more than 1000 exabytes in YEARS. (and int64 literally isn't large enough to handle too many exabytes. 128 bit processors when)
```

Comment:
Change this back to exbiytes. The correct term here is exbibytes.

Reply:
Sure.